### DISCUSSION TITLE
Computational speed of Anura3D

### DISCUSSION THREAD

**Discussion by Geozhengchen** [Anura3D/Anura3D_OpenSource#270] [Q&A]
Hello, members of the community! Recently, while using Anura3D to compute the impact problem, I noticed that after changing the dry soil in the original case to saturated soil, the computation speed significantly decreased. However, I checked the CPU usage on my computer, and it was only around 20%. How can I improve the computational speed of Anura3D?

**Comment by hjk1108** [comment]
I'm having the same issue. Has it been resolved?  How can I use CPU acceleration?
